src/yoroi-wallets/cardano/getTime.ts

```diff
@@ -0,0 +1,24 @@
+import {GENESIS_DATE} from './shelley/constants'
+
+export const getTime = (currentTimeMS: number) => {
```

Comment:
after the migration of networkInfo it needs to be based on the network (guild/preprod) have different parameters, for now, is fine